pkg/monitortests/authentication/requiredsccmonitortests/monitortest.go

```diff
@@ -135,7 +151,7 @@ func suggestSCC(pod *v1.Pod) string {
 		return fmt.Sprintf("suggested required-scc: '%s'", pod.Annotations[securityv1.ValidatedSCCAnnotation])
 	}
 
-	return "cannot suggest required-scc, validated SCC is custom"
+	return "cannot suggest required-scc, validated SCC is a custom/non-standard SCC"
```

Comment:
Since you can now distinguish between a custom vs non-standard SCC, use separate messages.